driver/csiplugin/controllerserver.go

```diff
@@ -2881,13 +2867,12 @@ func (cs *ScaleControllerServer) CreateNewSnapshot(ctx context.Context, conn con
 	return snapName, nil
 }
 
-
 func (cs *ScaleControllerServer) retryToCreateNewSnap(ctx context.Context, conn connectors.SpectrumScaleConnector, filesystemName, filesetName, snapName string) {
 
 	loggerId := utils.GetLoggerId(ctx)
 	klog.Infof("[%s] retrying to check whether new snapshot is required", loggerId)
 	for i := 0; i < retryCount; i++ {
-		time.Sleep( retryInterval * time.Second)
+		time.Sleep(retryInterval * time.Second)
 	}
 
 }
```

Comment:
Remove functions lockBucket and unlockBucket as they are not used now.

Reply:
done